doc/manual/manual.tex

```diff
@@ -5365,6 +5365,91 @@ \subsubsection{Tracking finite strain}
+
+To obtain the developer version of geomIO, you can clone the bitbucket repository by executing the command
+\begin{verbatim}
+ git clone https://bitbucket.org/geomio/geomio.git
+\end{verbatim}
+or you can download geomIO \href{https://bitbucket.org/geomio/geomio/downloads}{here}. 
+You will then need to add the geomIO source folders to your MATLAB path by running the file located in
+\texttt{/path/to/geomio/installation/InstallGeomIO.m}.
+An extensive documentation for how to use geomIO can be found \href{http://geomio-doc.bitbucket.org/}{here}.
+Among other things, it explains \href{http://geomio-doc.bitbucket.org/tuto2D.html#drawing}{how to generate drawings in Inkscape}
+that can be read in by geomIO, which involves assigning new attributes to paths in Inkscape's XML editor. 
+In particular, a new property `phase' has to be added to each path, and set to a value corresponding to the 
+index of the material that should be present in this region in the initial condition of the geodynamic model.   
+
+\note{geomIO currently only supports the latest stable version of Inkscape (0.91), and other versions might not
+  work with geomIO or cause errors. Moreover, geomIO currently does not support grouping paths (paths can still 
+  be combined using \texttt{Path$\rightarrow$Union}, \texttt{Path$\rightarrow$Difference} or similar commands), 
+  and only the outermost closed contour of a path will be considered. This means that i.e. for modelling a spherical 
+  annulus, you would have to draw two circles, and assign the inner one the same phase as the background of your drawing.}
+
+We will here use a drawing of a jellyfish located in \url{doc/manual/cookbooks/geomio/jellyfish.svg}, where different phases 
+have already been assigned to each path (Figure \ref{fig:jelly-picture}). 
+\begin{figure}[h!]
+    \centering
+    \includegraphics[width=0.2\textwidth]{cookbooks/geomio/jellyfish.pdf}
+    \caption{Vector drawing of a jellyfish.}
+    \label{fig:jelly-picture}
+\end{figure}
+\note{The page of your drawing in Inkscape should already have the extents (in px) that you later want to use in your model (in m).}
+
+After geomIO is initialized in MATLAB, we \href{http://geomio-doc.bitbucket.org/tuto2D.html#assigning-phase-to-markers}
+{run geomIO as desribed in the documentation}, loading the default options and then specifying all the option we want to 
+change, such as the path to the Input file, or the resolution. 
+\lstinputlisting[language=matlab]{cookbooks/geomio/run_geomio.part1.m}
+You can view all of the options available by typing \texttt{opt}. 
+
+In the next step we create the grid that is used for the coordinates in the ascii data initial conditions file 
+and assign a phase to each grid point: 
+\lstinputlisting[language=matlab]{cookbooks/geomio/run_geomio.part2.m}
+You can plot the \texttt{Phase} variable in MATLAB to see if the drawing was read in and all phases are assigned correctly 
+(Figure \ref{fig:jelly-plot}). 
+\begin{figure}[h!]
```

Comment:
We almost never use `[h]` or even `[h!]` because it messes up the whole page design if a figure doesn't actually fit at the current position. Just use `[tb]` -- you have the reference to the figure in the text above.